### PR TITLE
Add Edge Network and Edge Bridge repo links in the Important block for track API

### PR DESCRIPTION
**Pull request by cacheung**
Add Edge Network and Edge Identity repo links in the Important block for track API.
User should be able to know the usage info of trackAPI -  using Edge Network and Edge Bridge. 
Edge Bridge readme explains detailed info about trackAPIs, so we put Edge Bridge link before Edge Network.

Also add a note in acptarget table the migration doc. Using Analytics for Target, the user should use Edge Network and Optimize.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] I have signed the [Adobe Open Source CLA](http://opensource.adobe.com/cla.html).
- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: packages/core/README.md [hunk 1]
```diff
@@ -234,6 +234,9 @@ MobileCore.setLogLevel(LogLevel.VERBOSE);
 
 - #### trackAction
 
+> [!IMPORTANT]  
+> trackAction is supported through [Edge Bridge](../edgebridge) and [Edge Network](../edge) extensions. 
+
```
Comment (cacheung) [inline packages/core/README.md:239]: Should we call out Analytics Extension is not supported here?

**Comment by yangyansong-adbe** [comment]
@cacheung can you also update the API docs here https://github.com/adobe/aepsdk-react-native/blob/40def9d20a16ae6cccb96d3d0fc4aeda759e72d8/packages/core/ts/MobileCore.ts#L194C79-L194C79 
and here https://github.com/adobe/aepsdk-react-native/blob/40def9d20a16ae6cccb96d3d0fc4aeda759e72d8/packages/core/ts/MobileCore.ts#L209
Thanks!

File: packages/core/README.md (hunk 1 above)
Comment (emdobrin) [reply packages/core/README.md:239]: I think we should be good with covering that in the migration doc.

File: docs/migration.md [hunk 2]
```diff
@@ -27,10 +27,12 @@ At this time, the following ACP-prefix libraries can be switched out with their
 | @adobe/react-native-acpanalytics | NA, workflows supported through [Edge Network](../packages/edge) or [Edge Bridge](../packages/edgebridge) extension. |
 | @adobe/react-native-acpmedia | NA |
 | @adobe/react-native-acpaudience | NA |
-| @adobe/react-native-acptarget | @adobe/react-native-aeptarget |
+| @adobe/react-native-acptarget | @adobe/react-native-aeptarget, for using Analytics for Target, workflows supported through [Edge Network](../packages/edge) and [Optimize](../packages/optimize) extensions. |
```
Comment (emdobrin) [inline docs/migration.md:30]: can we split this on two lines, reading:
@adobe/react-native-aeptarget
For using Analytics for Target, workflow is supported through [Edge Network](../packages/edge) and [Optimize](../packages/optimize).

File: docs/migration.md [hunk 3]
```diff
@@ -27,10 +27,12 @@ At this time, the following ACP-prefix libraries can be switched out with their
...
 | @adobe/react-native-acpcampaign | NA |
 | @adobe/react-native-aepassurance:2.x (compatible with ACP libraries) | @adobe/react-native-aepassurance:4.x (compatible with AEP libraries)|
 
+> Note: Analytics library is not supported in AEP React Native. For implementing the Analytics workflow, you need to use Edge or Edge Bridge Extensions. Please refer to [migrate to Edge Network](https://developer.adobe.com/client-sdks/documentation/adobe-analytics/migrate-to-edge-network/) for more info.
```
Comment (emdobrin) [inline docs/migration.md:34]: ... Analytics workflow, register and configure the Edge Network or Edge Bridge libraries. ...

**Review by emdobrin — APPROVED** [review]

File: docs/migration.md [hunk 4]
```diff
@@ -24,13 +24,15 @@ At this time, the following ACP-prefix libraries can be switched out with their
 | @adobe/react-native-acpuserprofile | @adobe/react-native-aepuserprofile |
 | @adobe/react-native-acpplaces | @adobe/react-native-aepplaces |
 | @adobe/react-native-acpplaces-monitor | NA |
-| @adobe/react-native-acpanalytics | NA, workflows supported through [Edge Network](../packages/edge) or [Edge Bridge](../packages/edgebridge) extension. |
+| @adobe/react-native-acpanalytics | NA, Analytics workflows supported through [Edge Network](../packages/edge) or [Edge Bridge](../packages/edgebridge) extensions. |
 | @adobe/react-native-acpmedia | NA |
 | @adobe/react-native-acpaudience | NA |
-| @adobe/react-native-acptarget | @adobe/react-native-aeptarget |
+| @adobe/react-native-acptarget | @adobe/react-native-aeptarget, <br> Analytics for Target (A4T) is not supported.  If you want to use the A4T workflow,  it is supported through [Edge Network](../packages/edge) and [Optimize](../packages/optimize) extensions. |
```
Comment (yangyansong-adbe) [inline docs/migration.md:30]: Can we use the absolute path?

**Comment by yangyansong-adbe** [comment]
@cacheung If we want to release the core package to the npm registry, let's change the base branch to staging and update the core version in package.json.

**Review by yangyansong-adbe — APPROVED** [review]

**Pull request merged**